### ISSUE TITLE
Swift package doesn't work

### ISSUE THREAD

**Issue by aliak00** [aliak00/Tasker#14]
And I have no idea why.
When including in another project and building, you get this:
```
dyld: Library not loaded: @rpath/Tasker.framework/Tasker
  Referenced from: /Users/aliak/Library/Developer/CoreSimulator/Devices/52362EB4-894A-458D-8C7E-8922BDC98821/data/Containers/Bundle/Application/C042CE46-FC62-4239-A6D1-0090BFF54E10/flaneur.app/flaneur
  Reason: image not found
```

When you run `swift test` inside the root dir it's all good.

When you open Package.swift in xcode, for some reason the unit tests fail with:

```
2020-06-03 21:31:47.600120+0200 xctest[31601:974339] The bundle “TaskerTests.xctest” couldn’t be loaded because its executable couldn’t be located. Try reinstalling the bundle.
Program ended with exit code: 1
```

And the compilation errors are shown:

```
error: unable to open dependencies file (/Users/aliak/Library/Developer/Xcode/DerivedData/Tasker-ccymmsbfnrczrcfuvnbmrlkjysbs/Build/Intermediates.noindex/Tasker.build/Debug/Tasker.build/Objects-normal/x86_64/task-PACKAGE:/Users/aliak/dev/Tasker::MAINGROUP::REF_1::REF_2.d) (in target 'Tasker' from project 'Tasker')
Command CompileSwiftSources emitted errors but did not return a nonzero exit code to indicate failure
```

Also tried deleting DerivedData and cleaning project multiple times.

**Comment by ctreffs** [comment]
Maybe this could help - it did in another project for me: https://forums.swift.org/t/swift-packaged-version-of-my-library-cant-be-used-in-other-projects-xcode-cant-build/37201/2

**Comment by aliak00** [comment]
@ctreffs that was me :D and yeah it helped. Fixed it in commit #121a69483
